spec/models/events/creator_spec.rb

```diff
@@ -1,6 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe Events::Creator, :type => :model do
+
+  before do
+    Timecop.freeze(Time.local(1990))
```

Comment:
Is it better for this to be `Timecop.freeze`? Currently the spec is  setup to always run at the same point in time. Not really a big problem for this class, but not something I'd want new devs coming in to.